thriftpy2/rpc.py

```diff
@@ -7,7 +7,8 @@
 import urllib
 import warnings
 
-from thriftpy2._compat import PY35
+from thriftpy2.contrib.aio.rpc import make_client as make_aio_client  # noqa
+from thriftpy2.contrib.aio.rpc import make_server as make_aio_server  # noqa
```

Comment:
These two lines can be removed too.

```suggestion
```

Reply:
https://github.com/Thriftpy/thriftpy2/blob/61ef458059c63fb1fd0181780123de1f36462825/README.rst#L95-L101
https://github.com/Thriftpy/thriftpy2/blob/61ef458059c63fb1fd0181780123de1f36462825/examples/asyncio_echo/server.py#L2-L5
The ducument and example rely on this.